api/src/main/java/net/automatalib/ts/modal/transitions/GroupMemberEdge.java

```diff
@@ -0,0 +1,8 @@
+package net.automatalib.ts.modal.transitions;
+
+public interface GroupMemberEdge<G> {
+
+    G getGroup();
```

Comment:
Currently, I don't see this method used and at the only point where a `GroupMemberEdge` is used (`MMCVisualizationHelper`) it is used without a generic type parameter. I assume it is intended to associate complex objects with specific group member (ids) but wouldn't this fit better in an external mapping?

Reply:
At the time it seemed to be a more coherent and general concept than just defining an interface with memberIds. Currently, the group is always equivalent to the color attribute and therefore not essential.

Reply:
I'd vote for removing this feature then for now

Reply:
Ok